relay/relay.go

```diff
@@ -24,7 +24,7 @@ func UnmarshalKind(id graphql.ID) string {
 	if err != nil {
 		return ""
 	}
-	i := strings.IndexByte(string(s), ':')
+	i := strings.IndexByte(string(s),  ':')
```

Comment:
Accidental space here?

Reply:
It's intended. I am testing the CI. That's why it's at a draft stage :slightly_smiling_face:

Reply:
```
golangci-lint run ./...Failed in 
00:07
relay/relay.go:27: File is not `gofmt`-ed with `-s` (gofmt)00:07
	i := strings.IndexByte(string(s),  ':')00:07
internal/exec/subscribe.go:85:5: reflectvaluecompare: avoid using == with reflect.Value (govet)00:07
	if result == reflect.Zero(result.Type()) {00:07
	   ^00:07
graphql.go:200:5: reflectvaluecompare: avoid using == with reflect.Value (govet)00:07
	if s.res.Resolver == (reflect.Value{}) {00:07
	   ^00:07
subscriptions.go:24:5: reflectvaluecompare: avoid using == with reflect.Value (govet)00:07
	if s.res.Resolver == (reflect.Value{}) {00:07
	   ^00:07
internal/schema/schema_test.go:998:8: shadow: declaration of "err" shadows declaration at line 993 (govet)00:07
				if err := tt.validateError(err); err != nil {
```

The above is caught and there are some other issues too. I guess you have to fix it now.

Reply:
Yep, fixed those as well. My local tool was on a previous version.